Octokit/Models/Request/CollaboratorRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class CollaboratorRequest
+    {
+        public CollaboratorRequest()
+        {
+            Permission = Permission.Push;
```

Comment:
> So then we should have two overloads for Add and Invite in RepoCollaboratorsClient!? Because the docs say Only valid on organization-owned repositories.

I missed that in the docs.  Im confused whether it is saying the "permissions" attribute is only applicable to org repositories, or the whole "Add collaborator" endpoint.